src/main/java/com/jalasoft/jfc/model/utility/Md5Checksum.java

```diff
@@ -28,18 +29,18 @@
     private static final Logger logger = Logger.getLogger(Md5Checksum.class.getName());
 
     /**
-     *  This method return a boolean after to compare checksums.
-     * @param file receive the path of a file to get md5.
+     * Returns a boolean after to compare checksums.
```

Comment:
this comment is correct? "Returns a boolean after to compare checksums",  the correct way can be "Returns a boolean after to compare files' checksum

Reply:
no really because the client send just a Checksum throught postman